cypress/fixtures/work_orders/jobs.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "reference": "00012345",
```

Comment:
All your references in this spec are the same btw, I think there is a unique constraint on them so should be different